pyqtgraph/Qt.py

```diff
@@ -290,6 +290,59 @@ def pyqt5_qabort_override(*args, **kwds):
 if QT_LIB in [PYQT5, PYQT6, PYSIDE2, PYSIDE6]:
     # We're using Qt5 which has a different structure so we're going to use a shim to
     # recreate the Qt4 structure
+
+    if QT_LIB in [PYQT5, PYSIDE2]:
+        __QGraphicsItem_scale = QtWidgets.QGraphicsItem.scale	
+
+        def scale(self, *args):
+            warnings.warn(
+                "Deprecated Qt API, will be removed in 0.13.0.",
+                DeprecationWarning, stacklevel=2
+            )
+            if args:	
+                sx, sy = args	
+                tr = self.transform()	
+                tr.scale(sx, sy)	
+                self.setTransform(tr)	
+            else:	
+                return __QGraphicsItem_scale(self)
```

Comment:
should this return? weird that the other code path has no return but this does...

Reply:
Yes it does. ```scale()``` shadows a Qt5 _getter_ method.